api-module/src/main/java/org/devridge/api/domain/member/service/MemberService.java

```diff
@@ -0,0 +1,119 @@
+package org.devridge.api.domain.member.service;
+
+import lombok.RequiredArgsConstructor;
+import org.devridge.api.constant.MemberConstant;
+import org.devridge.api.constant.SkillConstants;
+import org.devridge.api.domain.member.dto.request.CreateMemberRequest;
+import org.devridge.api.domain.member.dto.request.DeleteMemberRequest;
+import org.devridge.api.domain.member.entity.Member;
+import org.devridge.api.domain.member.repository.MemberRepository;
+import org.devridge.api.domain.skill.entity.MemberSkill;
+import org.devridge.api.domain.skill.entity.MemberSkillId;
+import org.devridge.api.domain.skill.entity.Skill;
+import org.devridge.api.domain.skill.repository.MemberSkillRepository;
+import org.devridge.api.domain.skill.repository.SkillRepository;
+import org.devridge.api.exception.member.DuplEmailException;
+import org.devridge.api.exception.member.PasswordNotMatchException;
+import org.devridge.api.exception.member.SkillsNotValidException;
+import org.devridge.api.security.constant.SecurityConstant;
+import org.devridge.api.util.SecurityContextHolderUtil;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+@Service
+@RequiredArgsConstructor
+public class MemberService {
+    private final MemberRepository memberRepository;
```

Comment:
```suggestion

    private final MemberRepository memberRepository;
```
다른 클래스랑 컨벤션 맞춰서 클래스와 변수 사이에 개행 넣어주세요.

Reply:
넵! 수정했습니다.